client/src/app/components/CreateProject/styles.css

```diff
@@ -40,20 +40,30 @@
 }
 
 .form-wrapper button {
-  padding: 5px;
-  margin: 10px;
+  border: none;
+  height: 35px;
+  width: 75px;
+  padding: 8px;
+  margin: 2px;
+  background: #ec5252;
+  border: 1px solid #e8e9eb;
+  color: white;
+  border-radius: 3px;
+  font-size: 16px;
+  font-weight: bold;
+  cursor: pointer;
+  float: right;
```

Comment:
This is for aligning it to the right? I think that with float, the layout can be more difficult to reason about. And not sure, but the element dimensions that you declared above, will be ignored. 

For right positioning, there are other many alternatives. justify-text, position: relative, flex. These don't break the layout around the element.

Reply:
Modified to flex